test-java-lib/src/main/java/androidx/annotation/RecentlyNonNull.java

```diff
@@ -0,0 +1,14 @@
+package androidx.annotation;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.RetentionPolicy.CLASS;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/** stub for testing */
+@Retention(CLASS)
+@Target({METHOD, PARAMETER, FIELD})
+public @interface RecentlyNonNull {}
```

Comment:
Any reason why we can't depend on the actual annotation package for tests? I assume there is a version with just the annotations, right?

Reply:
From the discussion on #331 I thought these annotations got stripped out of the final jars, since they aren't supposed to be used in source code.  Do you have an idea of a library we could depend on that contains the annotations?

Reply:
Oh, I missed that, right it's not on the `androidx.annotations.*` jar.